app/assets/html/survey_questions.html

```diff
@@ -1,6 +1,6 @@
 <div id="survey-questions-container">
     <div class="header">{{ currentTopic }}</div>
-    <p id="description">{{ pageDescription() }}</p>
+    <p id="description">{{ numQuestions }} questions.</p>
```

Comment:
Change one of the conditional in pageDescription method in surveyQuestions.coffee instead. What you did here will completely break state knowledge.